### DISCUSSION TITLE
页面顶部和页面底部鼠标手势逻辑优化

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [shuax/MouseInc.Settings#161] [Ideas]
在设置里看这俩在浏览器中的鼠标手势对应动作是键入 <kbd>Home</kbd> 和 <kbd>End</kbd>，这会导致当焦点位于输入框时，无法滚动至页面顶部和页面底部。

希望能优化该逻辑，使焦点位于输入框是该手势仍能滚动至页面顶部和底部。

**Comment by shuax** [comment]
这个优化不了，只能装一些扩展指定对应的特殊快捷键这样就非常稳定了。

**Reply by Geeyun-JY3** [reply]
相关扩展是指哪些扩展？
